Dockerfile

```diff
@@ -3,5 +3,6 @@ FROM openjdk:11-jre-slim
 WORKDIR /app
 COPY docker /
 ENV MICRONAUT_CONFIG_FILES=/app/application.yml
+USER 10001
```

Comment:
I think it's better to create a real user, no ?

Reply:
It generally doesn't matter. Eg OpenShift just randomizes the user id a container is run as. The advantage of using a numeric ID (independent of whether the user actually exists, i.e. has a passwd entry), is that Kubernetes can tell if a container is run as a non-root user because the number is != 0 (see also https://stackoverflow.com/questions/49720308/kubernetes-podsecuritypolicy-set-to-runasnonroot-container-has-runasnonroot-and)

Reply:
Yep! I've built a custom image `FROM akhq` here: https://hub.docker.com/layers/chgl/akhq/0.15.0-rootless/images/sha256-daf750aa2e94e1c3a75b91faf5df08bdda4d6ef8dcf70cd8df7e02cc8b2acfd5?context=explore and it works fine. I'm currently running the official container image within a Kubernetes cluster but overriding the securityContext (after realizing the chart already provides this feature) to a non-root user. No problems with either approach, as the default permissions for config and jar allow read-access to any user.